gdb/riscv-tdep.h

```diff
@@ -69,10 +69,13 @@ enum {
 
 #define RISCV_LAST_REGNUM (RISCV_NUM_REGS - 1)
 
+typedef enum { UNKNOWN, YES, NO } supported_t;
```

Comment:
I don't feel great about relying on C++ so soon after the transition. How about:
```
typedef enum { SUP_UNKNOWN, SUP_YES, SUP_NO } supported_t;
```